nautobot_golden_config/tables.py

```diff
@@ -170,13 +168,10 @@ def __init__(self, *args, **kwargs):
             .values_list("rule__feature__slug", flat=True)
             .distinct()
         )
-        extra_columns = [(feature, ComplianceColumn(verbose_name=feature)) for feature in features]
-        kwargs["extra_columns"] = extra_columns
         # Nautobot's BaseTable.configurable_columns() only recognizes columns in self.base_columns,
         # so override the class's base_columns to include our additional columns as configurable.
-        self.base_columns = copy.deepcopy(self.base_columns)
-        for feature, column in extra_columns:
-            self.base_columns[feature] = column
+        for feature in features:
```

Comment:
Can you verify this change? I think it was deep copy for a reason, e.g. as you changed features there may be an issue.

Reply:
I think I checked it pretty thoroughly but let me know if there's anything else I should check. I went to the list view and made some changes to columns, reset columns, and opened the configure modal and clicked save without changing columns. In the comment that I linked to I documented quite a bit of research that was done before making this change.

Reply:
I also did the same checks against Nautobot v2.0.0

Reply:
probably best to speak to it. I am referring to creating new config features and config rules, and having them show up, **not** with the configure columns button.